### DISCUSSION TITLE
"Infinitely running" tasks

### DISCUSSION THREAD

**Discussion by cosmicBboy** [flyteorg/flyte#3463] [RFC Incubator]
This is a stub RFC for implementing a task type that can potentially run infinitely. The plan for this is that it will serve as a primitive building block for downstream applications such as reactive execution graphs.

**Comment by fg91** [comment]
Potentially also for model deployments?

**Comment by davidmirror-ops** [comment]
2023-11-09 Contributor's meetup notes: OK to close.
